cypress/support/fragments/settings/users/limits.js

```diff
@@ -26,6 +26,12 @@ export default {
       saveButton.click(),
     ]);
   },
+  setMaximumOutstandingFeeFineBalance: (balance) => {
+    cy.do([
+      TextField({ name:'cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a' }).fillIn(balance),
```

Comment:
can we do without hardcode "name" here?

Reply:
the structure of the DOM does not allow us to do this. in the future, when the developers fix these elements, I will change this interactor